### PR TITLE
Save Screenshot, upload it into GCS

### PR DESCRIPTION
**Pull request by wh1210**

### CODE REVIEW

**Comment by wh1210** [comment]
Please don't review it right now, I am doing some tests in GCS. Will let you know when ready~

**Comment by wh1210** [comment]
Now it is ready for review~

File: server/webdriver_tests/timeline_test.py [hunk 1]
```diff
@@ -67,6 +67,8 @@ def test_charts_from_url_directly_and_uncheck_statvar(self):
         charts = self.driver.find_elements_by_class_name("card")
         # Assert there are two charts.
         self.assertEqual(len(charts), 2)
+        self.driver.set_window_size(width=1000,height=2000,windowHandle='current')
+        self.driver.save_screenshot("test_screenshots/test_charts_from_url_directly_and_uncheck_statvar.png")
```
Comment (shifucun) [inline server/webdriver_tests/timeline_test.py:71]: For the screenshot, can you undo the save_screenshot() in this file, create a new file "screenshot_test".py and do the screenshot in there.

Basically this file will be a central place to do all the screenshot. Please define a list of urls there and make screenshot for each url. A starting point could be:

/place?dcid=country/USA

/place?dcid=country/USA&topic=Demographics

/place?dcid=country/USA&topic=Health

/tools/timeline#&place=geoId/0606000,geoId/2511000,geoId/2603000,geoId/1777005,geoId/1225175,geoId/4815976&statsVar=Median_Age_Person

/tools/timeline#&place=geoId/0606000,geoId/2511000,geoId/2603000,geoId/1777005,geoId/1225175,geoId/4815976&statsVar=Count_CriminalActivities_ViolentCrime

/tools/timeline#place=country%2FUSA%2CgeoId%2F06085&pc=1&statsVar=CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase

Comment (wh1210) [reply server/webdriver_tests/timeline_test.py:71]: Sure, SGTM

Comment (shifucun) [reply server/webdriver_tests/timeline_test.py:71]: Also can you name the saved file as "1_place_usa.png", "2_place_usa_demographics.png", ..., The name starts with the order we want them to appear in the screenshot page

Comment (wh1210) [reply server/webdriver_tests/timeline_test.py:71]: Good to know this!

File: server/webdriver_tests/screenshot_test.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+# Class to test timeline tool.
+class TestScreenShot(WebdriverBaseTest):
+
+    def test_pages_and_sreenshot(self):
+        """Test these page can show correctly and do screenshot."""
+        self.driver.set_window_size(width=1000,height=2000,windowHandle='current')
+
+        # Test TIMELINE_EXPLORER_URL_1.
```
Comment (shifucun) [inline server/webdriver_tests/screenshot_test.py:36]: can you make this a for loop by building an object at the beginning like:

[
...
{
'url': '/place?dcid=country/USA'
'filename_suffix": 'place_usa.png'
}
...
]

File: server/webdriver_tests/screenshot_test.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        self.driver.get(self.url_ + TIMELINE_EXPLORER_URL_1)
+        time.sleep(5)
+        charts = self.driver.find_elements_by_class_name("card")
+        # Assert there are one chart.
+        self.assertEqual(len(charts), 1)
+        self.driver.save_screenshot("test_screenshots/1_median_age_six_places.png")
```
Comment (shifucun) [inline server/webdriver_tests/screenshot_test.py:42]: please make "test_screenshots" a constant at the beginning of the file.

Comment (wh1210) [reply server/webdriver_tests/screenshot_test.py]: Done

File: server/webdriver_tests/screenshot_test.py (hunk 2 above)
Comment (wh1210) [reply server/webdriver_tests/screenshot_test.py]: Done

**Pull request merged**